monai/transforms/composables.py

```diff
@@ -555,7 +553,7 @@ def __call__(self, data):
 
 class RandAdjustContrastd(Randomizable, MapTransform):
     """
-    dictionary-based wrapper of RandAdjustContrast.
+    dictionary-based version RandAdjustContrast.
```

Comment:
if you change 'RandAdjustContrast' to ``` :py:class:`monai.transforms.transfroms.RandAdjustContrast` ``` it'll show a nice link in the doc webpage. (it's the same for the other transforms)

Reply:
changed for all the transforms in our codebase in the latest patch.
Thanks.